app/views/simple_calendar/_week_calendar.html.erb

```diff
@@ -16,15 +16,15 @@
       <% date_range.each_slice(7) do |week| %>
         <tr>
           <% week.each do |day| %>
-            <%= content_tag :td, class: calendar.td_classes_for(day) do %>
-              <% if defined?(Haml) && respond_to?(:block_is_haml?) && block_is_haml?(block) %>
-                <% capture_haml do %>
+              <%= content_tag :td, class: calendar.td_classes_for(day) do %>
+                <% if defined?(Haml) && respond_to?(:block_is_haml?) && block_is_haml?(block) %>
```

Comment:
This line is confusing for someone who isn't familiar with the library you're using so it may be a good idea to wrap it with a meaningful helper method